sos/plugins/openvswitch.py

```diff
@@ -41,6 +41,18 @@ def setup(self):
             "/run/openvswitch/ovsdb-server.pid",
             "/run/openvswitch/ovs-vswitchd.pid"
         ])
+        proc_files = ["environ", "cgroup", "maps", "numa_maps", "limits",
+                      "status"]
+        with open("/var/run/openvswitch/ovs-vswitchd.pid", 'r') as pidfile:
+            pid = pidfile.readline().splitlines()[0]
+            # List TID & thread command name.
+            self.add_copy_spec(["/proc/%s/task/*/%s"
+                                % (pid, pf) for pf in proc_files])
+
+        ps_Lo = "ps -p %s -Lo" % pid
+        ps_sched_opts = "pid,tid,user,pcpu,psr,trs,rss,size,sz,drs,vsz,"
+        ps_sched_opts += "maj_flt,min_flt,class,rtprio,ni,pri,stat,wchan:20,"
+        ps_sched_opts += "flags,comm"
```

Comment:
I think this is better served going into the `process` plugin.

We do 5 total `ps` calls at the moment, I'd say it's time for a review of those, especially with this new addition, to see where we can merge or drop certain collections. 

That review wouldn't be on you here. In any event however, I don't think adding these calls to individual plugins is the right approach.